modules/tooltip/react/stories/Tooltip.stories.mdx

```diff
@@ -118,7 +118,7 @@ white-space: nowrap;
 
 **Note**: Text truncation should be avoided if possible. A user should not have to activate a
 tooltip to access important content. If user-generated content is being truncated, the following
-situation might occur which is a bad user experence. Consider the following list:
+situation might occur which is a bad user experience. Consider the following list:
```

Comment:
I made sure to enable spell check in MDX files. Apparently that was the only file type not covered by my spell checker.